ironfish/src/account/accountsdb.ts

```diff
@@ -174,17 +175,7 @@ export class AccountsDB {
     await this.headHashes.clear()
   }
 
-  async replaceHeadHashes(map: Map<string, string>): Promise<void> {
```

Comment:
Wasn't sure we'd ever use this, so removing. Can re-add if we find a need for it.